staging/src/k8s.io/apiextensions-apiserver/pkg/controller/status/naming_controller.go

```diff
@@ -188,6 +188,7 @@ func (c *NamingConditionController) calculateNamesAndConditions(in *apiextension
 		namesAcceptedCondition.Status = apiextensions.ConditionTrue
 		namesAcceptedCondition.Reason = "NoConflicts"
 		namesAcceptedCondition.Message = "no conflicts found"
+		namesAcceptedCondition.LastTransitionTime = metav1.NewTime(time.Now())
```

Comment:
With all the condition changes in this area, would it make sense to add a check and a timestamp update to the apiextensions.SetCondition func? Haven't looked into that yet, maybe I am completely wrong.

Reply:
Commented here https://github.com/kubernetes/kubernetes/issues/54148#issuecomment-337662313.